package.json

```diff
@@ -84,18 +85,11 @@
 		"viewport-list": "^5.0.1"
 	},
 	"devDependencies": {
+		"@types/node": "^10.12.0",
```

Comment:
Don't forget to import types for all the node modules. See: https://github.com/sindresorhus/pageres/pull/342/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R88